src/bin/cratesfyi.rs

```diff
@@ -231,12 +235,15 @@ pub fn main() {
             let conn = db::connect_db().expect("failed to connect to the database");
             db::delete_crate(&conn, &name).expect("failed to delete the crate");
         }
+
```

Comment:
Is there a reason you added this whitespace?

Reply:
This improved the readability significantly for me. The if-else structures feels very dense and I found it difficult to parse the different top-level subcommands. By adding a whitespace the commands `delete-crate`, `start-web-server`, `daemon` and `queue` stand out from each other more.
Since it doesn't touch any previously written code I thought it would okay to add. But in the end, it's mostly personal preference so I don't mind reverting it as well.

Reply:
I don't have a strong opinion so if it makes it easier for you we can leave it in.